asm/istio/options/openshift.yaml

```diff
@@ -0,0 +1,38 @@
+# Copyright 2020 Google LLC
```

Comment:
s/2020/2022

Reply:
sorry my bad, I directly copied it. Will change it.